src/routes/plans/+page.svelte

```diff
@@ -422,43 +428,66 @@
             />
           </Field>
 
-          <fieldset>
-            <DatePickerField
-              field={startTimeDoyField}
-              label="Start Time - YYYY-DDDThh:mm:ss"
-              name="start-time"
-              on:change={onStartTimeChanged}
-              on:keydown={updateDurationString}
-              use={[
-                [
-                  permissionHandler,
-                  {
-                    hasPermission: canCreate,
-                    permissionError,
-                  },
-                ],
-              ]}
-            />
-          </fieldset>
-
-          <fieldset>
-            <DatePickerField
-              field={endTimeDoyField}
-              label="End Time - YYYY-DDDThh:mm:ss"
-              name="end-time"
-              on:change={updateDurationString}
-              on:keydown={updateDurationString}
-              use={[
-                [
-                  permissionHandler,
-                  {
-                    hasPermission: canCreate,
-                    permissionError,
-                  },
-                ],
-              ]}
-            />
-          </fieldset>
+          {#if $plugins.time.enableDatePicker}
+            <!-- TODO not sure if the on:keydown actually exists in DatePickerField -->
```

Comment:
Currently, it doesn't exist in both the `DatePicker` and `DatePickerField`. `keydown` is explicitly being used in `DatePicker` for autocomplete purposes, but is not propagated up.